src/NuGet.Services.EndToEnd/Support/Clients/SimpleHttpClient.cs

```diff
@@ -35,10 +35,15 @@ internal SimpleHttpClient()
 
         public async Task<T> GetJsonAsync<T>(string url, ITestOutputHelper logger)
         {
-            return await GetJsonAsync<T>(url, allowNotFound: false, logger: logger);
+            return await GetJsonAsync<T>(url, allowNotFound: false, logResponseBody: true, logger: logger);
         }
 
         public async Task<T> GetJsonAsync<T>(string url, bool allowNotFound, ITestOutputHelper logger)
+        {
+            return await GetJsonAsync<T>(url, allowNotFound, logResponseBody: true, logger: logger);
+        }
```

Comment:
How did you determine which json responses we should log?

Reply:
Polling requests do not log their response.

Everything else does.